cmd/tracee-ebpf/initialize/kernelconfig.go

```diff
@@ -0,0 +1,28 @@
+package initialize
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/aquasecurity/libbpfgo/helpers"
+	"github.com/aquasecurity/tracee/cmd/tracee-ebpf/internal/debug"
+)
+
+func KernelConfig() (*helpers.KernelConfig, error) {
+	kernelConfig, err := helpers.InitKernelConfig()
+	if err != nil && debug.Enabled() {
+		fmt.Fprintf(os.Stderr, "KConfig: warning: could not check enabled kconfig features\n(%v)\n", err)
+		fmt.Fprintf(os.Stderr, "KConfig: warning: assuming kconfig values, might have unexpected behavior\n")
+	}
+
+	// do not fail (yet ?) if we cannot init kconfig
+	kernelConfig.AddNeeded(helpers.CONFIG_BPF, helpers.BUILTIN)
```

Comment:
In case `err != nil` above you will still run these commands, but `kernelConfig` failed to initialize

Reply:
Ahah, I was misled by the comment but I see the original code doesn't continue. So just adding a `return config, nil` should be fine there?